site/docs/4.1/components/carousel.md

```diff
@@ -287,7 +285,7 @@ Options can be passed via data attributes or JavaScript. For data attributes, ap
       <td>touch</td>
       <td>boolean</td>
       <td>true</td>
-      <td>Whether the carousel should handle touch event and allow swipe left/right.</td>
+      <td>Whether the carousel should handle touch events and allow swipe left/right (only for mobile browsers).</td>
```

Comment:
no, not really "only mobile browsers" - there are desktop computers with touch (e.g. surface). also, hammer uses pointer events where available, so technically not "touch events". suggest changing it to what i said originally in my follow-up comment :)

Reply:
Oh ok I'll change that. We removed HammerJS we use a Vanilla JS solution

Reply:
oh, missed that you're using vanilla JS (as i saw the hammer simulator being loaded as well). in which case could I'd say it would be good to also support pointer events...but that's probably a more involved addition...

Reply:
for reference: supporting pointer events will also make this work on touch-enabled desktop/laptop devices in Edge (which, by default, does not expose touch events by default, but does support touch interactions via pointer events)